ddb-dm-screen.user.js

```diff
@@ -7,246 +7,201 @@
-                                    case 'intelligence-score': character.stats.int.value = y.value; break;
-                                    case 'wisdom-score': character.stats.wis.value = y.value; break;
-                                    case 'charisma-score': character.stats.cha.value = y.value; break;
-                                }
-                            };
-                        };
-                        for (j=0; j < mods.length; j++) {
-                            let y = mods[j];
-                            if (y.type == "proficiency") {
-                                switch(y.subType) {
-                                    case 'strength-saving-throws': character.stats.str.saveProficiency = 1; break;
-                                    case 'dexterity-saving-throws': character.stats.dex.saveProficiency = true; break;
-                                    case 'constitution-saving-throws': character.stats.con.saveProficiency = true; break;
-                                    case 'intelligence-saving-throws': character.stats.int.saveProficiency = true; break;
-                                    case 'wisdom-saving-throws': character.stats.wis.saveProficiency = true; break;
-                                    case 'charisma-saving-throws': character.stats.cha.saveProficiency = true; break;
-                                    default:
-                                        if (typeof(character.skills[y.subType]) === "undefined" ) {
-                                            character.skills[y.subType] = new Skill(y.subType, character.proficiency, "proficiency");
-                                        } else {character.skills[y.subType].setProf("proficiency")}; break;
-                                };
-                            };
-                            if (y.type == "expertise") {
-                                if (typeof(character.skills[y.subType]) === "undefined" ) {
-                                    character.skills[y.subType] = new Skill(y.subType, character.proficiency, "expertise");
-                                } else {character.skills[y.subType].setProf("expertise")};
-                            };
-                            if (y.id == "classFeature_270_1439") {
-                                character.stats.str.saveBonus += parseInt(character.stats.cha.bonus());
-                                character.stats.dex.saveBonus += parseInt(character.stats.cha.bonus());
-                                character.stats.con.saveBonus += parseInt(character.stats.cha.bonus());
-                                character.stats.int.saveBonus += parseInt(character.stats.cha.bonus());
-                                character.stats.wis.saveBonus += parseInt(character.stats.cha.bonus());
-                                character.stats.cha.saveBonus += parseInt(character.stats.cha.bonus());
-                            };
```

Comment:
I realize this part has been removed, but just a curiosity question...  Why was 'cha' bonus added to everything in this block?  Was that a bug?

Reply:
No, it was 6th level paladin aura that gave everyone including paladin his charisma bonus to all saves.

Reply:
The thing I kinda rushed the pull request before you start adding even more stuff and everything gets harder to change. So yeah, I didn't take the time today to account for all the skills logic you did since they weren't displayed in the table. That's why most of it is removed.